### DISCUSSION TITLE
Registration email contact name should be changed

### DISCUSSION THREAD

**Discussion by jonas-martinez** [lenra-io/lenra_cli#106] [General]
![image](https://user-images.githubusercontent.com/36544012/175897768-00bec56b-22a5-43a8-accc-c3c3906181a0.png)
 
As you can see on this screenshot, I received the registration email but the name of the sender (contact@lenra.io) is `contact` which is not really good. Maybe we should rename to `lenra` or the full `contact@lenra.io`

**Comment by taorepoara** [comment]
I think just `Lenra`  should be good

**Comment by pichoemr** [comment]
Resolved in https://github.com/lenra-io/server/pull/335
